app/Models/Theme.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace App\Models;
+
+use A17\Twill\Models\Behaviors\HasMedias;
+use A17\Twill\Models\Behaviors\HasPosition;
+use A17\Twill\Models\Behaviors\HasRevisions;
+use A17\Twill\Models\Behaviors\HasTranslation;
+use A17\Twill\Models\Behaviors\Sortable;
+use A17\Twill\Models\Model;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class Theme extends Model implements Sortable
+{
+    use HasFactory, HasMedias, HasPosition, HasRevisions, HasTranslation;
+
+    protected $fillable = [
+        'published',
+        'position',
+        'title',
+        'intro',
+    ];
+
+    public $translatedAttributes = [
+        'title',
+        'intro',
+    ];
+
+    public $mediasParams = [
+        'icon' => [
+            'default' => [
+                [
+                    'name' => 'default',
+                    'ratio' => 1,
+                    'minValues' => [
+                        'width' => 76,
+                        'height' => 76,
```

Comment:
In the `ThemeRequest` validation rules, the min height/width is 75.

Reply:
Good catch. I've updated these to be the same as the validation rules and made a note to address this duplication.